cpp/include/cudf/detail/hashing.hpp

```diff
@@ -37,9 +37,26 @@ std::pair<std::unique_ptr<table>, std::vector<size_type>> hash_partition(
  * @param stream CUDA stream used for device memory operations and kernel launches.
  */
 std::unique_ptr<column> hash(table_view const& input,
+                             hash_id hash_function                     = hash_id::HASH_MURMUR3,
                              std::vector<uint32_t> const& initial_hash = {},
                              rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource(),
                              cudaStream_t stream                 = 0);
 
+std::unique_ptr<column> identity_hash(
```

Comment:
what is this function?

Reply:
Would be the wrapper call for the IdentityHash function, I did not end up implementing it and will remove it.